### PR TITLE
Update BattleOfDazaralor esEs Locales

### PR DESCRIPTION
**Pull request by sabinico**
- Updated all locales from raid Battle of Dazar'alor to Spanish

### CODE REVIEW

File: BattleOfDazaralor/Locales/esES.lua [hunk 1]
```diff
@@ -1,4 +1,4 @@
-local L = BigWigs:NewBossLocale("Battle of Dazar'alor Trash", "esES") or BigWigs:NewBossLocale("Battle of Dazar'alor Trash", "esMX")
+local L = BigWigs:NewBossLocale("Basura de Batalla de Dazar'alor", "esES") or BigWigs:NewBossLocale("Basura de Batalla de Dazar'alor", "esMX")
```
Comment (funkydude) [inline BattleOfDazaralor/Locales/esES.lua]: You're not supposed to change these names. They are the module names.

**Comment by funkydude** [comment]
Thank you. Are you planning to translate everything else to Spanish also?

File: BattleOfDazaralor/Locales/esES.lua (hunk 1 above)
Comment (sabinico) [reply BattleOfDazaralor/Locales/esES.lua]: Oops! Sorry I will fix that tomorrow!

**Comment by sabinico** [comment]
Yep! I will try to translate everything to Spanish. 😊

**Comment by funkydude** [comment]
I fixed the names and merged it. Looking forward to seeing the rest translated!

**Pull request merged**